charts/tailscale-operator/templates/operator-role.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
The operator Role defines the necessary access for controlling resources related to the proxy StatefulSet and backing Secrets.  This will only ever be scoped to the namespace in which the operator is deployed, at least as it is designed today.  That could change if the operator adds support for placing proxy pods in the same namespaces as the Service/Ingress that it targets.